mkdocs.yml

```diff
@@ -1,29 +1,30 @@
-site_name: LCLS Users Documentation
-repo_url: https://github.com/lcls-users/lcls-users
-site_author: FP, AP, GFD
-copyright: © 2023 LCLS
-markdown_extensions:
-    - admonition
-    - attr_list
-    - def_list
-nav:
-    - Before your experiment:
-        - Computing Accounts and Resources: 'before/accounts.md'
-        - Routine beamtime analysis: 'before/analysis.md'
-    - During your experiment:
-        - Running the timetool: 'during/timetool.md'
-    - After your experiment:
-        - Reanalyzing timing data: 'after/timetool.md'
-    - Miscellaneous:
-        - Troubleshooting: 'troubleshooting.md'
-        - Glossary: 'glossary.md'
-theme:
-    name: readthedocs
-    highlightjs: true
-    hljs_languages:
-        - yaml
-        - python
-        - bash
-    collapse_navigation: false
-    navigation_depth: 3
-    titles_only: true
+site_name: LCLS Users Documentation
+repo_url: https://github.com/lcls-users/lcls-users
+site_author: FP, AP, GFD
+copyright: © 2023 LCLS
+markdown_extensions:
+    - admonition
+    - attr_list
+    - def_list
+nav:
+    - Before your experiment:
+        - Computing Accounts and Resources: 'before/accounts.md'
+        - Routine beamtime analysis: 'before/analysis.md'
+    - During your experiment:
+        - Running the timetool: 'during/timetool.md'
+    - After your experiment:
+        - Reanalyzing timing data: 'after/timetool.md'
+    - Miscellaneous:
+        - Computers101: 'tutorials/computers101.md'

```

Comment:
I think we should move this up! How about the first item under "Before your experiment"? :D